src/OpenLoco/Vehicles/Vehicle.cpp

```diff
@@ -18,6 +18,31 @@ using namespace OpenLoco::Literals;
 
 namespace OpenLoco::Vehicles
 {
+#pragma pack(push, 1)
+    // There are some common elements in the vehicle components at various offsets these can be accessed via vehicle_base
+    struct VehicleCommon : vehicle_base
```

Comment:
I'm assuming this was moved here to prevent using this anywhere else?

Reply:
Yes, i want all common access through the vehicle base.